### PR TITLE
small fixes

### LINKED ISSUE #3111: [BUG]: check_metric import from pycaret.utils not working
### pycaret version checks

- [X] I have checked that this issue has not already been reported [here](https://github.com/pycaret/pycaret/issues).

- [X] I have confirmed this bug exists on the [latest version](https://github.com/pycaret/pycaret/releases) of pycaret.

- [X] I have confirmed this bug exists on the master branch of pycaret (pip install -U git+https://github.com/pycaret/pycaret.git@master).

### Issue Description

This command doesn't work any more in PyCaret 3.

```
from pycaret.utils import check_metric
```

Instead we have to do this:

```
from pycaret.utils.generic import check_metric
```

Expected Fix: `check_metric` should be available on root `pycaret.utils`.

### Reproducible Example

```python
see above.
```

### Expected Behavior

see above.

### Actual Results

```python-traceback
see above.
```

### Installed Versions

master

### LINKED ISSUE #3112: [BUG]: ValueError: Mime type rendering requires nbformat>=4.2.0 but it is not installed
### pycaret version checks

- [X] I have checked that this issue has not already been reported [here](https://github.com/pycaret/pycaret/issues).

- [X] I have confirmed this bug exists on the [latest version](https://github.com/pycaret/pycaret/releases) of pycaret.

- [X] I have confirmed this bug exists on the master branch of pycaret (pip install -U git+https://github.com/pycaret/pycaret.git@master).

### Issue Description

When using `plot_model` in multiple modules, I get this error:

```
ValueError: Mime type rendering requires nbformat>=4.2.0 but it is not installed
```

By updating the library manually, every thing seems to work fine.

Expected Fix: Add `nbformat>=4.2.0` in requirements.txt.

### Reproducible Example

```python
see above.
```

### Expected Behavior

see above.

### Actual Results

```python-traceback
see above.
```

### Installed Versions

master

### PR DESCRIPTION
**Pull request by moezali1**
# Related Issue or bug

Info about Issue or bug

Closes #3111 
Closes #3112

# Describe the changes you've made

A clear and concise description of what you have done to successfully close your assigned issue. Any new files? or anything you feel to let us know!

# Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. 

# Describe if there is any unusual behaviour of your code(Write `NA` if there isn't)

A clear and concise description of it.

# Checklist:

- [ ] My code follows the style guidelines of this project.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] Any dependent changes have been merged and published in downstream modules.

# Screenshots

 Original           | Updated
 :--------------------: |:--------------------:
 **original screenshot**  | <b>updated screenshot </b> |

### CODE REVIEW

**Comment by ngupta23** [comment]
Could you fix the code quality issue. Just run 

> isort filename.py

**Comment by moezali1** [comment]
@ngupta23 I have only changed requirements.txt and init.py. why does it still require formatting?

**Comment by tvdboom** [comment]
The problem is in the ordering of the imports. You need to download the `isort` package and run `isort pycaret` in the terminal. That should fix it.

File: .gitignore [hunk 1]
```diff
@@ -35,3 +35,4 @@ dask*/
 /.venv
 /.devcontainer
 tmp/
+nbtest1.ipynb
```
Comment (ngupta23) [inline .gitignore:38]: Why is this necessary in the gitignore file?

Comment (tvdboom) [reply .gitignore:38]: indeed it's better to exclude test notebooks from git, instead of ignoring them @moezali1

**Review by tvdboom — APPROVED** [review]
